Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js

```diff
@@ -161,9 +161,6 @@ export default function Homepage() {
       <StudentAccess/>
       <EthisimIntro/>
 
-      <Box className={classes.copyright}>
-        <Copyright />
-      </Box>
     </div>
```

Comment:
Sorry for being strict but keep the copyright on the homepage XD

Reply:
lmao aight